.env.example

```diff
@@ -34,6 +34,9 @@ ALLOWED_TELEGRAM_USER_IDS=USER_ID_1,USER_ID_2
 # TEMPERATURE=1.0
 # PRESENCE_PENALTY=0.0
 # FREQUENCY_PENALTY=0.0
+# IMAGE_MODEL=dall-e-3
+# IMAGE_QUALITY=hd
+# IMAGE_STYLE=natural
 # IMAGE_SIZE=512x512
```

Comment:
You suggest dalle-3 by default, which cannot handle 512x512. If you uncomment it, it will return an error.

Reply:
thanks, I changed the IMAGE_SIZE suggestion in the example env to 1024x1024
The default values are dall-e-2 and 512x512 set in the main.py